rust/core-lib/src/tabs.rs

```diff
@@ -741,6 +795,41 @@ impl CoreState {
         }
     }
 
+    #[cfg(feature = "notify")]
+    fn handle_open_file_tail_fs_event(&mut self, event: Event) {
+        use notify::event::*;
+
+        let path = match event.kind {
+            EventKind::Modify(ModifyKind::Any) => &event.paths[0],
+            other => {
+                debug!("Ignoring event in open file {:?}", other);
+                return;
+            }
+        };
+
+        let buffer_id = match self.file_manager.get_editor(path) {
+            Some(id) => id,
+            None => return,
+        };
+
+        let has_changes = self.file_manager.check_file(path, buffer_id);
+        let is_pristine = self.editors.get(&buffer_id).map(|ed| ed.borrow().is_pristine()).unwrap();
```

Comment:
Again, unwrapping isn't nice (unless None is a programmer error in which case the called code should probably `panic!` anyway)

Reply:
Agree but this code is copied as is from original.

Reply:
Hm, I guess this returning None doesn't really happen then? @cmyr